org.phoenicis.playonlinux.json

```diff
@@ -0,0 +1,81 @@
+{
+  "app-id": "org.phoenicis.playonlinux",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "add-build-extensions": {
```

Comment:
Isn't `add-extensions` enough here?

Reply:
I'm not sure. I had this discussion here: https://github.com/flatpak/flatpak-builder/issues/223. My final understanding was that I should rather use "add-build-extensions".

Reply:
I am really not sure if the compat libs should be added as build-extensions. Try changing it to `add-extensions`.

Reply:
It's updated.

Reply:
@barthalion Any idea why the 32-bit compat extensions are not pulled when installing the Flatpak?